### PR TITLE
Add Clipper tutorial to repo

### PR DESCRIPTION
**Pull request by RehanSD**
Please open port 3000 and 9090 on the EC2 instance

### CODE REVIEW

**Comment by simon-mo** [comment]
@RehanSD can you remove clipper/tutorial/notebook-images/.DS_Store

**Comment by shaneknapp** [comment]
.gitignore ftw

**Review by simon-mo — APPROVED** [review]

**Comment by simon-mo** [comment]
Because we modified the master `Makefile.inc` I want to hold off merging on this until @jey takes a took and approves.

**Comment by jey** [comment]
Thanks for the ping; taking a look now

File: Makefile.inc
Comment (jey) [inline Makefile.inc]: Could we do this part by instead adding the following to `clipper/Makefile`?
```
DOCKER_RUN_FLAGS += --network host -v /tmp:/tmp
```

Also, do we really need `-v /tmp:/tmp`, or is that an artifact from a merge conflict? (cf. 54be3a3d6f7f254b145788b8082ea8013b07fb45)

**Review by jey — CHANGES_REQUESTED** [review]
Everything looks good overall, but I think the `Makefile` changes can be kept to `clipper/`. Can you try that out and report back?

File: Makefile.inc
Comment (simon-mo) [reply Makefile.inc]: Yes we need it. Clipper uses the tmp folder to store artifacts that will be passed to docker daemon. I'll address the comment and merge.

**Pull request merged**
